readme.md

```diff
@@ -158,7 +160,7 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Question:** The `.sign()` method takes three arguments. Describe each argument in your own words, using the above code as an example.
 
-* **Your Answer:**
+* **Your Answer:** The payload, secret for signing, and options like expiration date
```

Comment:
-The first argument `payload` represents the data to be passed in the JWT,
The second argument is the secret, in this case `MYSECRETPASSCODE`. 
The last argument `options` can include a number of different options such as expiresIn, algorithm, audience, issuer, etc.